### DISCUSSION TITLE
Support for adding a comment to entire table

### DISCUSSION THREAD

**Discussion by ManojKiranA** [laravel/framework#37683] [Ideas] [answered]
By default laravel support to add comment for columns in table. But there is no way to add comment to table. It would be nice to have out of the box support for it

[See](https://stackoverflow.com/questions/37493431/how-to-add-comment-to-table-not-column-in-laravel-5-migration) Stackoverflow issues here

**Comment by chuckrincon** [comment]
Hi there ✋🏻 !

Unfortunately the [PR](https://github.com/laravel/framework/pull/35539) that support this feature, implemented for MySQL and PostgreSQL, was rejected

**Comment by shaedrich** [comment, marked as answer]
This was implemented in 9.x via #42401

**Comment by ManojKiranA** [comment]
Thanks for the heads up

**Reply by shaedrich** [reply]
You're welcome
